recipe/meta.yaml

```diff
@@ -12,16 +12,14 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 0
+  number: 1
+  noarch: python
   script: python setup.py install --single-version-externally-managed --record record.txt
```

Comment:
Can this be rewritten as
```
script: {{ PYTHON }} -m pip install . --no-deps -vv
```

(possibly change requirements.host setuptools --> pip)?

Reply:
no-deps will not work as we need MDA.

Reply:
Isn't that already defined under `run`?

Reply:
I changed it to use pip.